### ISSUE TITLE
Linked image resize handlers do not refresh when converting block image to inline

### ISSUE THREAD

**Issue by oleq** [ckeditor/ckeditor5#9521]
## 📝 Provide detailed reproduction steps (if any)

![](https://user-images.githubusercontent.com/1099479/115390777-f2029800-a1de-11eb-86b3-00e294d03cab.gif)

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by oleq** [comment]
This is caused by a listener in `WidgetResize`

```js
viewSelection.on( 'change', () => {
	const selectedElement = viewSelection.getSelectedElement();

	this.visibleResizer = this.getResizerByViewElement( selectedElement ) || null;
} );
```

which depends on `viewSelection.getSelectedElement()`, which fails when a linked inline image is selected because the selection is on the link attribute element, but not on an image

![image.png](https://images.zenhubusercontent.com/5db82ed39f933c00013cf856/4bf20968-ba01-4561-aa2f-538059c2e0aa)

This is not the first time when we face it (https://github.com/ckeditor/ckeditor5/blame/i/2052-inline-images/packages/ckeditor5-image/src/imageutils.js#L132-L135) but this time it's going to be hard to address because we cannot use image-specific logic here.

**Comment by oleq** [comment]
Closed in #9526.
